src/pages/api/server.py

```diff
@@ -116,7 +116,7 @@ async def add_attendance_record(student_id: int, class_id: int, timestamp):
 
     cur = conn.cursor()
     cur.execute(
-        f"INSERT INTO attendance (classid, studentid, entry_timestamp) VALUES ('{class_id}', '{student_id}', '{timestamp}');"
+        f"INSERT INTO attendance (attendanceid, classid, studentid, entry_timestamp) VALUES ('{attendanceid}', '{class_id}', '{student_id}', '{timestamp}');"
```

Comment:
We shouldn't need to manually add the attendanceID. The ID should auto increment based on the db. The risk with manually adding it instead of using the db is that we may enter an ID with the same ID